face-to-face/face-to-face.schema.json

```diff
@@ -64,10 +68,15 @@
               "type": "string"
             },
             "i": {
-              "description": "Identifier that issuee controls, according to issuer",
+              "description": "AID that issuee controls, according to issuer",
```

Comment:
I deliberately left this as Identifier, because I imagine you could issue this to someone that doesn't control an AID. The issuer must have an AID, but the issuee only should have one. They could have a DID instead.

Reply:
updated as per comment